controller/channel.js

```diff
@@ -0,0 +1,105 @@
+let router = require('express').Router();
+let uuid   = require('uuid/v4');
+let Event  = require('events');
+let val    = require('validator');
+
+router.get('/channel/:id', (req, res) =>
+{
+    if
+    (
+        val.isUUID(req.params.id, 4) &&
+        global.channels.hasOwnProperty(req.params.id)
+    )
+    {
+        let the_channel = global.channels[req.params.id];
```

Comment:
what is global?

Reply:
Using the global variable to save cache of each of channel's data and all users connected to each channel within it. This is to not constantly have to query database for each connection and only query database when video starts or ends, channel created or removed, or if any data is missing.